migrations/15-add_third_party_fields.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports.id = 'add_third_party_fields';
+
+const thirdPartyText = 'Operated By The Triton Engineering Student Council.'
+const organisedBy = 'TESC'
+module.exports.up = function (done) {
+    var events = this.db.collection('events');
+    events.updateMany({}, {$set: {thirdPartyText, organisedBy}}, done);
+};
+
+module.exports.down = function(done) {
+    var events = this.db.collection('events');
+    events.update({}, {$unset: {thirdPartyText: 1, organisedBy: 1}}, done);
+}
```

Comment:
If thirdPartyText is undefined then we can assume it is a TESC event. Therefore, it doesn't need to be in this migration.